gr-filter/lib/qa_mmse_fir_interpolator_ff.cc

```diff
@@ -61,7 +59,7 @@ namespace gr {
 	  float expected = test_fcn((i + 3) + imu * inv_nsteps);
 	  float actual = intr.interpolate(&input[i], imu * inv_nsteps);
 
-	  CPPUNIT_ASSERT_DOUBLES_EQUAL(expected, actual, 0.004);
+	  BOOST_CHECK(std::abs(expected - actual) <= 0.004);
 	  // printf ("%9.6f  %9.6f  %9.6f\n", expected, actual, expected - actual);
```

Comment:
A comment, not a request for change in your patch:

Generally, when fundamentally touching code, don't hesitate to get rid of such commented out lines of code that don't contribute anything in terms of e.g. illustrating how something was meant to be used.